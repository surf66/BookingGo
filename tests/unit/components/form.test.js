import React from 'react';
import Form from '../../../src/js/components/form';

test('should match snapshot', () => {
  const component = shallow(<Form />);

  expect(component).toMatchSnapshot();
});