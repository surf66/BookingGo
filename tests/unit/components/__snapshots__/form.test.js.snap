// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<section>
  <form
    className="pickup-form"
  >
    <h1>
      Let's find your ideal car
    </h1>
    <Connect(SearchField) />
    <Connect(SuggestionList) />
  </form>
</section>
`;
